content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,134 @@
+* Toute conduite qui pourrait être raisonnablement considérée comme inappropriée
+  dans le milieu professionnel
+
+## Nos responsabilités
+
+Les dirigeant•e•s de la communauté sont chargé•e•s de clarifier et de faire respecter nos normes de
+comportements acceptables et prendront des mesures correctives appropriées et équitables en
+réponse à tout comportement qu'ils ou elles jugent inapproprié, menaçant, offensant ou nuisible.
+
+Les dirigeant•e•s de la communauté ont le droit et la responsabilité de supprimer, modifier ou rejeter
+les commentaires, les contributions, le code, les modifications de wikis, les rapports d'incidents ou de bogues et autres contributions qui
+ne sont pas alignés sur ce Code de Conduite et communiqueront les raisons de la modération
+le cas échéant.
+
+## Portée d'application
+
+Ce Code de Conduite s'applique à la fois au sein des espaces du projet ainsi que
+dans les espaces publics lorsqu'un individu représente officiellement le projet ou sa
+communauté. Font parties des exemples de représentation d'un projet ou d'une
+communauté le fait d'utiliser une adresse email propre au projet, de poster sur
+les réseaux sociaux avec un compte officiel ou d'intervenir pour représenter le
+projet au cours d'un évènement en ligne ou hors-ligne.
+
+## Application
+
+Les cas de comportements abusifs, harcelants ou tout autre comportement
+inacceptables peuvent être signalés en contactant l'équipe du projet à
+[INSÉRER UNE ADRESSE EMAIL]. 
+Toutes les plaintes seront examinées et feront l'objet d'une enquête rapide et équitable.
+
+Tous les dirigeants de communauté sont tenus de respecter la vie privée et la sécurité des
+personnes ayant signalées un incident.
+
+## Directives d'application
+
+Les dirigeants de communauté suivront ces lignes directrices sur l'Impact Communautaire afin de déterminer
+les conséquences de toute action qu'ils jugent contraire au présent Code de Conduite :
+
+### 1. Correction
+
+**Impact Communautaire**: Utilisation d'un langage inapproprié ou tout autre comportement jugé
+non professionnel ou non bienvenu dans la communauté.
+
+**Conséquence**: Un avertissement privé et écrit des chefs de communauté, fournissant
+une clarification sur la nature de la violation et une explication des raisons pour lesquelles
+le comportement était inapproprié. Des excuses publiques peuvent être demandées.
+
+### 2. Avertissement
+
+**Impact Communautaire**: Une violation par un seul incident ou une série d'actions.
+
+**Conséquence**: Un avertissement avec des conséquences dû à la poursuite du comportement.
+Une non-interaction avec les personnes concernées, y compris l'interaction non sollicitée avec
+ceux qui appliquent le Code de Conduite, pendant une période déterminée. 
+Cela comprend le fait d'éviter les interactions dans les espaces communautaires ainsi que sur les canaux externes
+comme les médias sociaux. Le non-respect de ces conditions peut entraîner

```

Comment:
Either are good, I heard "non-respect" too, not sure if its an anglicisme.

Reply:
You’re right. Both are correct.

Reply:
Do you have any preference ?

Reply:
Non, just use either one consistently. Might be worth considering whether "violation" is triggering to victims of sexual assault and avoid it in that case.

Reply:
Indeed, it could be a good idea to avoid those kind of words, sorry about that.
Do you think that "non-respect" would be more suitable ?

Reply:
Yeah, it works too. I try to avoid "non-" prefix in general, as there's usually a more correct word to say the same thing. Here it is "violation," but it is kind of a weird word in French, so maybe go back to using "non-respect."